api_tests/lib/ledgers/ledger_runner.ts

```diff
@@ -25,15 +23,19 @@ export class LedgerRunner {
 
     constructor(
         private readonly projectRoot: string,
-        private readonly logDir: string
+        private readonly logDir: string,
+        private global: HarnessGlobal
```

Comment:
This is weird, why can't we just use the global `global` var?

Reply:
Unfortunately global is not yet available as it is only introduced outside in the TestEnvironment. 
I'll follow your proposal from below and rename.